gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -236,10 +236,18 @@ private static String projectIdFromUri(String path) throws IOException {
         String[] argEntry = arg.split("=");
         switch (argEntry[0]) {
           case "fields":
-            // List fields are in the form "projects(field1, field2, ...)"
-            options.put(
-                "fields",
-                argEntry[1].substring("projects(".length(), argEntry[1].length() - 1).split(","));
+            // List fields are in the form "projects(field1, field2, ...),nextPageToken"
+            String option = argEntry[1];
+            int projectStart = option.indexOf("projects(");
+            int projectEnd = option.indexOf(')');
+            if (projectStart != -1 && projectEnd != -1
```

Comment:
Yes, the idea was to make it more generic in case we ever decide to add some strange field selector that does not specify project fields